kalite/distributed/features/steps/superuser_create.py

```diff
@@ -5,6 +5,10 @@
 
 modal_container = "superusercreate-container"
 
+@given("I go to homepage")
```

Comment:
You don't need to define a new step here, considering that `@given("I am on the homepage")` already exists and does the same thing. If you change the wording of your .feature file to match that step, it should use it automatically.